nox/command.py

```diff
@@ -57,16 +101,16 @@ def which(program: str, paths: list[str] | None) -> str:
     raise CommandFailed(f"Program {program} not found")
 
 
-def _clean_env(env: dict[str, str] | None) -> dict[str, str] | None:
-    if env is None:
-        return None
-
+def _clean_env(env: dict[str, str] | None) -> dict[str, str]:
     clean_env = {}
 
-    # Ensure systemroot is passed down, otherwise Windows will explode.
-    clean_env["SYSTEMROOT"] = os.environ.get("SYSTEMROOT", "")
+    for var in _ENV_VARS_TO_ALWAYS_INCLUDE:
+        if var in os.environ:
```

Comment:
I think this has the potential to be a race as written (possible the environment var is there when checked on the first check, but not on assignment).